test/middleware/CompanyAuthMiddleware.test.ts

```diff
@@ -78,6 +79,33 @@ describe('Company Authentication Middleware', () => {
 
     });
 
+    it('should redirect to penalty details page no appeal data in session', async () => {
+
+        const encryptionService = createSubstituteOf<JwtEncryptionService>(service => {
+            service.encrypt(Arg.any()).resolves('');
+        });
+
+        const sessionStore = Substitute.for<SessionStore>();
+        const companyAuthMiddleware = new CompanyAuthMiddleware(
+            sessionStore,
+            encryptionService,
+            companyAuthConfig,
+            sessionStoreForAuthConfig,
+            featureFlag
+        );
+
+        const nextFunction = createSubstituteOf<NextFunction>();
+        const response = createSubstituteOf<Response>();
+        const request = getRequestSubstitute({}, '');
+
+        const startingWithRedirectUrl = (redirect : string) => redirect.startsWith(PENALTY_DETAILS_PAGE_URI);
```

Comment:
What is that? Looks like arg matcher. Var names does not help. Since it is used only in one place, why not to inline it to brush the problem under the carpet.

Reply:
updated this